G/Graphviz/build_tarballs.jl

```diff
@@ -0,0 +1,66 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Graphviz"
+version = v"2.40.1"
+
+# Collection of sources required to complete build
+sources = [
+    "https://graphviz.gitlab.io/pub/graphviz/stable/SOURCES/graphviz.tar.gz" =>
+    "ca5218fade0204d59947126c38439f432853543b0818d9d728c589dfe7f3a421",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd graphviz-2.40.1/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+
+# For some reason the build wasn't able to pick up that the system provided a
+# `sys/stat.h` file, so lib/sfio/sfsetbuf.c redefines `struct stat` which is an
+# error. So we manually set the preprocessor macro definition to prevent that.
+# (Presumably only some of the linux systems provide it, otherwise it wouldn't
+# be configurable, so we probably need to enable this iff it's missing.)
+if [[ "${target}" == *-linux* ]]; then  # TODO: figure out what this should be
+    make CFLAGS="-D_sys_stat=1" -j${nproc}
```

Comment:
Minor nitpick: the way I usually do this is by adding
```sh
if [[ "${target}" == *-linux* ]]; then
    export CFLAGS="-D_sys_stat=1"
fi
```
before `./configure`, in this way you don't need to do anything special in the `make` call

Reply:
Hmm, yeah i tried doing that, but it didn't seem to pick it up.. i wasn't really sure why. :'( We can look into trying that again if we can get it to work! ✔️

Reply:
Are you sure you tried exactly this?  Doing this before `./configure` is important, because it will add the custom `CFLAGS` directly into the `Makefile`, and also `export` is useful to make sure the subprocess will read the value of the variable.

Reply:
Huh, no i'm not sure. I _thought_ i tried exactly that (copied from a different PR of yours), but i looks like it worked for you now so 🤷‍♂!